src/textual/widgets/_data_table.py

```diff
@@ -3,7 +3,9 @@
 import functools
 from dataclasses import dataclass
 from itertools import chain, zip_longest
+from multiprocessing import Event
```

Comment:
If you have some kind of auto-import adder in your IDE. I think you should turn it off. Pretty sure we don't use multiprocessing here.

Reply:
I _also_ don't know how this ended up here! 😅 
I _might_ have some auto-import thing turned on.

Reply:
> I might have some auto-import thing turned on.

I *think* it's part of pylance/pyright; it ha a habit of doing the same thing via eglot in Emacs. Generally it only does the import if you pick an auto-imported completion (in Emacs it's generally easy to see where it's comping from so you can avoid it), and normally I noticed if it's managed to sneak one in when it comes to reviewing the commit in magit (atomic commits for the win here, of course).

What I've yet to find is how to tell pylance/pyright to turn it off. There is apparently a setting (`python.analysis.autoImportCompletions`) but I've yet to figure out a non-project-specific way of configuring it.